HLTrigger/JetMET/interface/AlphaT.h

```diff
@@ -29,21 +29,12 @@ class AlphaT {
 };
 
 
-// -----------------------------------------------------------------------------
```

Comment:
Because the constructor does not have to look different now for pointer types: `std::men_fn` is basically the generalization of `std::mem_fun` and `std::men_fun_ref`.

From https://en.cppreference.com/w/cpp/utility/functional/mem_fn:
    Both references and pointers (including smart pointers) to an object can be used when invoking a std::mem_fn. "

I hope I understood this correctly...